x-pack/legacy/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -65,3 +94,21 @@ export const MonitorPage = () => {
     </>
   );
 };
+
+const mapStateToProps: MapStateToPropsParam<StateProps, {}, AppState> = state => ({
+  selectedMonitor: selectSelectedMonitor(state),
+});
+
+const mapDispatchToProps: MapDispatchToPropsFunction<DispatchProps, {}> = (dispatch, own) => {
+  return {
+    dispatchGetMonitorStatus: (monitorId: string) => {
+      dispatch(
+        getSelectedMonitor({
+          monitorId,
+        })
+      );
+    },
+  };
+};
+
+export const MonitorPage = connect(mapStateToProps, mapDispatchToProps)(MonitorPageComponent);
```

Comment:
Can you move this part of component to components/connected/pages? we are trying to keep all the data injecting in same folder. It helps us in testing presentational components.

Reply:
After thinking about it, since we've decided to reorganize this all soon, are you OK with it being left here for this PR?